nautobot/core/tests/test_models.py

```diff
@@ -61,3 +68,12 @@ def test__content_type_caching_disabled(self):
             self.FakeBaseModel._content_type_cached
             self.FakeBaseModel._content_type_cached
             self.assertEqual(mock__content_type.call_count, 2)
+
+    @skipUnlessDBFeature("supports_json_field")
```

Comment:
Have you run this test locally just to confirm that it isn't being skipped? :-)

Reply:
Yes, confirm locally it was not being skipped.